### ISSUE TITLE
when i add an image to the toastview it animates the image

### ISSUE THREAD

**Issue by jordanZeleny** [BastiaanJansen/toast-swift#59]
When i add a image to the toastview, it works well, except that the image seems to animate larger after the toast was presented. 

also, there should be a variable for size of the image

**Comment by wyattharrell** [comment]
Seeing this after building with iOS 18

**Comment by jisu15-kim** [comment]
Could you show me more detail resource about this issue?
Perhaps some images or GIFs? 
I wasn't able to reproduce it when testing on iOS 18.
@jordanZeleny @wyattharrell

**Comment by wyattharrell** [comment]
@jisu15-kim Previous to iOS 18, the SF Symbol shown here would have been static 
https://github.com/user-attachments/assets/e234424e-34a0-4f17-b6ca-b0e88e796ade

**Comment by jisu15-kim** [comment]
@wyattharrell 
Thanks !
But could you provide more information how you calling this library? (usage code)

**Comment by wyattharrell** [comment]
Sure, here is how I am calling it:
```
private var toast: Toast?
```


```
func showToast() {
      toast?.close()
      
      let attributes = [
          NSAttributedString.Key.font: UIFont.rounded(ofSize: 17, weight: .medium),
      ]
      let attributedTitle  = NSMutableAttributedString(string: title, attributes: attributes)
      
      let config = UIImage.SymbolConfiguration(hierarchicalColor: .systemGreen)
      let image = UIImage(systemName: "checkmark.circle.fill")?.applyingSymbolConfiguration(config)
      
      toast = Toast.default(
                  image: image,
                  imageTint: nil,
                  title: attributedTitle,
                  viewConfig: .init(titleNumberOfLines: 0),
                  config: .init(direction: .top)
              )
      toast?.show(haptic: .success)
}
```

**Comment by jisu15-kim** [comment]
I've tested this in this vanilla environment, but I couldn't reproduce the issue. 
Could you please check for any external factors related to the library?

```
import UIKit
import Toast

class ViewController: UIViewController {
    
    private var toast: Toast?

    override func viewDidLoad() {
        super.viewDidLoad()   
    }

    @IBAction func buttonDidTapped(_ sender: Any) {
        toast?.close()

        let attributes = [
            NSAttributedString.Key.font: UIFont.systemFont(ofSize: 17, weight: .medium),
        ]
        let attributedTitle = NSMutableAttributedString(string: "Success", attributes: attributes)
        
        let config = UIImage.SymbolConfiguration(hierarchicalColor: .systemGreen)
        let image = UIImage(systemName: "checkmark.circle.fill")?.applyingSymbolConfiguration(config)
        
        toast = Toast.default(
            image: image!,
            imageTint: nil,
            title: attributedTitle,
            viewConfig: .init(titleNumberOfLines: 0),
            config: .init(direction: .top)
        )
        toast?.show(haptic: .success)
    }
}
```

(tested in Xcode 16, iOS18, iPhone 16 Pro, Swift6)

**Comment by BastiaanJansen** [comment]
Fixed in version 2.1.3 thanks to @jisu15-kim!
